windows/installer.nsi

```diff
@@ -212,6 +218,30 @@ Section "Install_64Bit" INST_64
                 MessageBox MB_OKCANCEL|MB_ICONEXCLAMATION "$(OLD_VER_REMOVE_FAILED)" /sd IDCANCEL IDOK continueinstall
                 goto quit
             ${EndIf}
```

Comment:
This code will only be executed if we find jamulus in the 32 bit prog. files folder and the current install is 64 bit.